.github/CODEOWNERS

```
@@ -140,3 +140,6 @@
 /lib/test.js @nodejs/test_runner
 /lib/internal/main/test_runner.js @nodejs/test_runner
 /lib/internal/test_runner/* @nodejs/test_runner
+
+# Single Executable Applications
+/src/node_sea* @nodejs/single-executable
```

Here's the full list:
```suggestion
/deps/postject @nodejs/single-executable
/doc/api/single-executable-applications.md @nodejs/single-executable
/doc/contributing/maintaining-postject.md @nodejs/single-executable
/doc/contributing/maintaining-single-executable-application-support.md @nodejs/single-executable
/src/node_sea* @nodejs/single-executable
/test/fixtures/postject-copy @nodejs/single-executable
/test/parallel/test-single-executable-* @nodejs/single-executable
/tools/dep_updaters/update-postject.sh @nodejs/single-executable
```